src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/services/services.component.ts

```diff
@@ -42,9 +46,14 @@ export class ServicesComponent extends ListWithDetails implements OnChanges, OnI
   // Do not display these columns
   @Input() hiddenColumns: string[] = [];
 
+  @Input()
+  clusterCreation = false;
```

Comment:
While this is a 'cosmetic' issue I think it's worthy to mention. The Services component shouldn't care less about anything called Cluster expansion wizard. That's a higher level workflow comprising several other components, while the Service component was born before any Cluster expansion workflow was even discussed.

It's the Cluster Expansion Component which should properly set the `hasDetails` section of the Services component.

Reply:
Done. Thanks